Lecture5.tex

```diff
@@ -122,8 +122,8 @@ \section{The Fixed-Policy Value Function}\label{s:FP_VF}
 \begin{itemize}
   \item The proof provides an explicit bound on $|{V^\pi }(s) - {V_n}{\kern 1pt} (s)|$. It may be seen that the convergence is exponential, with rate $O({\gamma ^n})$.
   \item Using vector notation, it may be seen that
-          \[{V_n}{\kern 1pt}  = {r^\pi } + {P^\pi }{r^\pi } +  \ldots  + {({P^\pi })^{n - 1}}{r^\pi } + {({P^\pi })^n}{V_0} = \sum\limits_{t = 0}^{n - 1} {{{({P^\pi })}^t}{r^\pi }}  + {({P^\pi })^n}{V_0}.\]
-Similarly,    ${V^\pi } = \sum\limits_{t = 0}^\infty  {{{({P^\pi })}^t}{r^\pi }}$.
+          \[{V_n}{\kern 1pt}  = {r^\pi } + {\gamma}{P^\pi }{r^\pi } +  \ldots  + {({\gamma}{P^\pi })^{n - 1}}{r^\pi } + {({\gamma}{P^\pi })^n}{V_0} = \sum\limits_{t = 0}^{n - 1} {{{({\gamma}{P^\pi })}^t}{r^\pi }}  + {({\gamma}{P^\pi })^n}{V_0}.\]
+Similarly,    ${V^\pi } = \sum\limits_{t = 0}^\infty  {{{({\gamma}{P^\pi })}^t}{r^\pi }}$.
```

Comment:
would be clearer to separate the exponent of gamma and P, i.e., \gamma^t {P^\pi}^t

Reply:
Hi Aviv @avivt , I've separated the exponent as you suggested

Reply:
what I meant is \gamma^t {P^\pi}^t instead of (\gamma {P^\pi})^t

Reply:
@avivt I removed the parenthesis: commit ebf9eb5edc1031c1cd049a6d853d3a676adcd7d9